### DISCUSSION TITLE
How to understand _abandoned

### DISCUSSION THREAD

**Discussion by unihon** [apache/skywalking#11784] [Q&A] [answered]
![image](https://github.com/apache/skywalking/assets/42078581/0e2894c1-0b99-4725-9d9f-6835d47f6f46)

If my service have 10 endpoints, If one of the qps is very high, will it be classified as `_abandoned`?

In fact, I don’t want it to be classified as _abandoned. Can I specify through configuration which APIs (endpoints) can be ignored and not counted?

**Comment by wu-sheng** [comment]
It is not 10, read the pull request if you want to understand more.

**Comment by wu-sheng** [comment, marked as answer]
It is determined by https://github.com/apache/skywalking/blob/af4f469078f3c4053221e7a232ad2f7e7f9b29db/oap-server/server-starter/src/main/resources/application.yml#L138C32-L138C32

The default value is 3000.

Don't guess for things. Look into codes, that is the point of open source.
